src/main/java/io/openshift/launchpad/ui/booster/MissionControlValidator.java

```diff
@@ -40,6 +40,14 @@
       });
    }
 
+   public List<String> getProjects(UIContext context)
```

Comment:
The implementation should cache the result in the UIContext attribute Map to avoid multiple remote invocations

Reply:
See the other methods impl